ui/src/apollo/queries.js

```diff
@@ -155,4 +191,10 @@ const getProjectByName = (apollo, name, ecosystemId) => {
   return response;
 };
 
-export { getEcosystems, GetBasicProjectInfo, getProjects, getProjectByName };
+export {
+  getEcosystems,
+  getEcosystemByID,
+  GetBasicProjectInfo,
```

Comment:
Just a minor comment, shouldn't this be `getBasicProjectInfo`?

Reply:
You're right, I've renamed it.